sxs/utilities/lvcnr/conversion.py

```diff
@@ -145,9 +145,14 @@ def convert(self, sxs_data_path, out_path, truncation_time=None, resolution=None
                                 truncation_time=truncation_time, resolution=resolution))
         log("Starting at "+time.strftime('%H:%M%p %Z on %b %d, %Y'))
 
-        # Load metadata.json from this simulation
-        with open(os.path.join(sxs_data_path, "metadata.json"), 'r') as f:
-            metadata = json.load(f)
+        # Load metadata from this simulation
+        try:
+            metadata = sxs.Metadata.from_file(os.path.join(sxs_data_path, "metadata.txt"))
+        except:
+            try:
+                metadata = sxs.Metadata.from_file(os.path.join(sxs_data_path, "metadata.json"))
+            except:
+                raise ValueError('Cannot load metadata.')
```

Comment:
This would prefer `metadata.txt`, but I think it makes more sense to prefer `metadata.json`, in which case the `from_file` function chooses automatically:

```suggestion
        try:
            metadata = sxs.Metadata.from_file(os.path.join(sxs_data_path, "metadata"))
        except:
            raise ValueError('Cannot load metadata.')
```